### PR TITLE
Develop 1.0.0

### PR DESCRIPTION
**Pull request by greymd**
Changes
* Update README.md
  + Put TravicCI batch
  + Brush sentences and sample codes up

* Options
  + Show version (`-V`, `--version`)
  + Show help (`-h`, `--help`)

* Test
  + Add test cases for new options.

### CODE REVIEW

File: ttcp.sh [hunk 1]
```diff
@@ -1,5 +1,9 @@
 #!/bin/bash
 
+# Version number of commands.
+# MAJOR.MINOR.PATCH
+readonly TTCP_VERSION="0.0.0"
```
Comment (greymd) [inline ttcp.sh:5]: After this PR is merged, let us update this line to `1.0.0` as a new commit.

Comment (mecab) [reply ttcp.sh:5]: 👍

File: README.md [hunk 2]
```diff
@@ -25,28 +28,38 @@ zplug "greymd/ttcopy"
...
+```sh
+$ git clone https://github.com/greymd/ttcopy.git ~/ttcopy
+```
+
+#### 2. Edit `.bashrc` or `.zshrc`
+
+And add following lines.
+
 ```sh
 # Set ID and PASSWORD **AS YOU LIKE**.
```
Comment (mecab) [inline README.md]: Well, though not scope in this commit, perhaps user will get confused to set identity. We should note

 1. User do not require to register. (the reason of why they set choose ID/Password "as they like")
 2. So the user should try to keep the id unique.

Can you add some lines?

**Review by mecab — COMMENTED** [review]
Nice work, especially on parsing args.

If you have time, can you add some notes about the identity to README?

File: README.md (hunk 2 above)
Comment (greymd) [reply README.md]: thx. I'll work on it.
README.md should be more updated (not only the part you pointed out).

**Review by mecab — APPROVED** [review]
🎉

**Comment by greymd** [comment]
I edited `README.md`  until I become satisfied.

**Review by mecab — COMMENTED** [review]
very cool image.

File: README.md [hunk 3]
```diff
@@ -1,52 +1,83 @@
...
 
-# Dependencies
+## Dependent commands
   * openssl
   * curl
 
 # Install
+First of all, please prepare ID and Password **as you like**.
+Do not be lazy. You are **NOT** required to register them on any services.
+Same data source (like clipboard) is shared within the hosts having same ID/Password.
```
Comment (mecab) [inline README.md]: I feel "Same data source" is a bit vague. How about "The data you copied can be pasted within the hosts having same ID/Password"

**Review by mecab — CHANGES_REQUESTED** [review]

**Comment by greymd** [comment]
@mecab thanks!! I don't mind if you make changes to this branch **directly** 😄

**Review by mecab — APPROVED** [review]

**Comment by mecab** [comment]
Merge?

**Comment by greymd** [comment]
Ok for me

**Comment by mecab** [comment]
🎉

**Comment by greymd** [comment]
🎉

**Pull request merged**